lib/streamlit/util.py

```diff
@@ -451,6 +451,32 @@ def print_url(title, url):
     click.secho(url, bold=True)
 
 
+def process_gitblob_url(url):
+    """Checks url to see if it describes a github gist "blob" URL.  
+    If so, returns a new URL to get the "raw" script. 
+    If not, returns URL unchanged.
+    """
```

Comment:
Nit: docstrings should follow https://www.python.org/dev/peps/pep-0257/ and https://numpydoc.readthedocs.io/en/latest/format.html

In particular:

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. 

i.e. add a newline between 455 and 456

> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".

i.e. Remove the "s" from "Checks".

Also, capitalize Github and Gist.